composer.json

```
@@ -25,6 +25,7 @@
     "require": {
         "php": ">= 7.1",
         "ext-json": "*",
+        "nesbot/carbon": "^1.26.0 || ^2.17",
```

Although Carbon should be a dependency for most if not all Laravel projects, i would like to avoid adding it as a dependency. Can we do without it?

---

We can ofcourse leave it out, but since we use Carbon (Tracing, @cache directive & Date(Time) scalar) I though it might be a good practice to do mention it as a direct dependency.

But we never had it before so I'm fine with leaving it out since we can be almost 100% sure Carbon is available.

---

Right, we are even using it in core parts of Lighthouse. We should loosen the constraints a bit, though, to match what the corresponding Laravel versions minimally need.

```suggestion
        "nesbot/carbon": "^1.26.0 || ^2.0",
```

---

Seems reasonable, we don't have a real constraint on the version 👍